programs/graph_program/src/state/connection.rs

```diff
@@ -1,17 +1,37 @@
-use anchor_lang::prelude::*;
+use anchor_lang::{prelude::*};
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, PartialEq, Eq)]
+pub enum ConnectionStatus {
+    Connected = 1,
+    Disconnected = 2,
+}
 
 #[account]
 #[derive(Default)]
 pub struct Connection {
     pub from: Pubkey,
     pub to: Pubkey,
+    // V2 Items.
+    pub status: Option<ConnectionStatus>, // No connection status equals connected
```

Comment:
```suggestion
```

I'm in favor dropping the status. Can be inferred from the connected_at and disconnected_at timestamps.

Reply:
Alright, dropping status